File: simpleclient.pl
#!/usr/bin/perl -w
use strict;

use Statgrabber;

my $timer = Statgrabber::start('simpleclient elapsed time');

foreach (0..4) {
	Statgrabber::count('foo');
}

foreach (0..4) {
	Statgrabber::average('bar',$_);
}

foreach (0..4) {
	# Test spaces, also
	Statgrabber::accumulate('b a z',$_);
}

sleep(1);

$timer->finish();

File: simpleclient.py
#!/usr/bin/python

import Statgrabber
import time

timer = Statgrabber.start('simpleclient elapsed time')

for x in range(0,5):
	Statgrabber.count('foo')

for x in range(0,5):
	Statgrabber.average('bar',x)

for x in range(0,5):
	# Test spaces, also
	Statgrabber.accumulate('b a z',x)

time.sleep(1.0)
timer.finish()
